go.mod

```diff
@@ -1,40 +1,44 @@
 module github.com/openshift/oadp-operator
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
 
 require (
 	github.com/aws/aws-sdk-go v1.44.253
-	github.com/go-logr/logr v1.2.4
+	github.com/go-logr/logr v1.3.0
 	github.com/google/uuid v1.3.0
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.11.0
-	github.com/onsi/gomega v1.27.8
-	github.com/openshift/api v0.0.0-20230213134911-7ba313770556 // release-4.12
+	github.com/onsi/ginkgo/v2 v2.13.0
+	github.com/onsi/gomega v1.29.0
+	github.com/openshift/api v0.0.0-20231218131639-7a5aa77cc72d // release-4.12
 	github.com/operator-framework/api v0.10.7
 	github.com/operator-framework/operator-lib v0.9.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.51.2
 	github.com/sirupsen/logrus v1.9.0
-	k8s.io/api v0.25.6
-	k8s.io/apiextensions-apiserver v0.24.2
-	k8s.io/apimachinery v0.25.6
-	k8s.io/client-go v0.25.6
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
+	k8s.io/api v0.29.0
+	k8s.io/apiextensions-apiserver v0.29.0
+	k8s.io/apimachinery v0.29.0
+	k8s.io/client-go v0.29.0
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
 require (
```

Comment:
remove this sectioning of packages?

Reply:
https://github.com/openshift/oadp-operator/pull/1264#issuecomment-1881538939